symplyphysics/laws/kinematics/kinetic_energy_of_body_from_mass_and_velocity.py

```diff
@@ -0,0 +1,22 @@
+from symplyphysics import (
```

Comment:
Does it matter that it is a kinetic energy of body? I think removing 'body' from the law name does not change the description, but shorter.